src/components/Layout/Header/NavBar/LanguageDropdown.tsx

```diff
@@ -0,0 +1,31 @@
+import { Menu, MenuButton, MenuItem, MenuList } from '@chakra-ui/menu'
+import { IconButton } from '@chakra-ui/react'
+import React from 'react'
+import { FaGlobe } from 'react-icons/fa'
+import { Text } from 'components/Text'
+
+export const LanguageDropdown = () => {
+  // TO-DO: Learn and implement Reacts State Management to save user Locale
+  const setLocale = React.useState('en')
```

Comment:
We should try to use the browser's language if it's available. There are locale APIs to get the current locale.

Reply:
`window.navigator.userLanguage` and `window.navigator.language` should cover ~98% of the browsers. I agree that the default locale should be pulled from the API, but there is definitely a need for users to be able to chance the locale. I'll be revisiting this in a few days -- I made the draft PR in hopes that someone would pick it up as it is currently not in my scope of responsibilities, but if no one picks it up then I'll gladly finish it when I have some time freeing up.